bgpd/bgp_evpn_vty.c

```diff
@@ -6971,6 +6971,7 @@ DEFUN(bgp_evpn_ead_es_rt, bgp_evpn_ead_es_rt_cmd,
 					  ecomadd))
 		bgp_evpn_mh_config_ead_export_rt(bgp, ecomadd, false);
 
+	ecommunity_free(&ecomadd);
```

Comment:
Shouldn't this be guarded by else statement?

Reply:
yes. fixed